src/projects.md

```diff
@@ -1,23 +1,102 @@
+
+<div id="projects-decommissioned">
+{% for project in site.data.decommissioned_projects %}
+  {% include project.html %}
+{% endfor %}
+</div>
+
+---
+
+{% include linkedHeading.html heading="Becoming an OpenOakland project" level=2 %}
+
+In response to member feedback that has resulted in an ongoing effort to address issues of equity and inclusion in our work, OpenOakland is reevaluating how our projects are vetted, adopted, and developed. This is an ongoing pilot that we continue to iterate on in an effort to ensure that projects serve their intended communities, consider potential unintended consequences, and foster greater inclusion of community voices—particularly those from underrepresented and underserved Oakland communities.
+
+If you have a new idea for an OpenOakland project:
+
+1. **Fill out the [Project Brief](https://docs.google.com/document/d/1k24P9JiAUEzJLPFRDjVh7aRZexax6NUhfPFLSI3R80M/edit?usp=sharing)**. We encourage you to join our [Slack workspace](http://slack.openoakland.org/) and share your draft with our membership, so we can collaborate together as you develop your idea.
+
+2. **Submit your draft brief to the #leadership channel** on Slack for formal consideration. Provided your brief is submitted at least two weeks in advance, it will be reviewed at the next Steering Committee meeting (a group of elected leadership and existing project reps), and you'll get some initial feedback and be asked to make adjustments accordingly.
+
+3. **Make any requested adjustments** based on the Steering Committee's feedback and resubmit the final brief.
+
+Once your final brief is submitted, the Steering Committee will hold a formal vote to approve or decline the project.
+
+### What makes a good project?
+
+We generally consider the following types of projects:
+
+- **Civic tech projects:** providing tools or services to Oaklanders or public agencies to increase access to and understanding of government.
+- **Events:** major events that require a team to execute.
+- **OpenOakland sustainability projects:** efforts to improve and sustain OpenOakland as an organization.
+
+Projects must demonstrate alignment to OpenOakland’s mission and values. Some ways a project might do so include:
+
+- Partnering with organizations to serve as domain experts in the needs of the community it serves
+- Forming a project team which has lived experience with the issue the project is focused on
+- Conducting user research to understand the needs of the community the project serves
+
+---
+
+{% include linkedHeading.html heading="Providing feedback" level=2 %}
```

Comment:
This section is still good placed on the bottom of the page. I think it works well as a outro

Reply:
Kept it with the "Becoming..." section, as it's probably more important info than, say decommissioned projects etc.